staging/src/k8s.io/client-go/discovery/cached/disk/round_tripper.go

```diff
@@ -22,11 +22,12 @@ import (
 	"path/filepath"
 
 	"github.com/gregjones/httpcache"
-	"github.com/gregjones/httpcache/diskcache"
 	"github.com/peterbourgon/diskv"
 	"k8s.io/klog/v2"
 )
 
+const maxDiskCacheSize = 100 * 1024 * 1024 // 100 MB
```

Comment:
Do you have an opinion that value? @Jefftree who also knows what a typical openapi v3 file size might be.

Reply:
BTW--this is the size for entire http cache (not just openapi files). The cache is attached to the `cacheRoundTripper` assigned to the discovery client. So the size must encompass all files the discovery client downloads. They usually end up under: `${HOME}/.kube/cache/http/<MD5_HASHED_URL_FILENAME>`

Reply:
Most OpenAPI V3 files for built-in types are between 100 and 200kB

Reply:
Yeah, I checked my `$HOME/.kube/cache` and it's 19mb, Sean's was much much bigger (way over 100mb). I think 100mb is actually pretty reasonable.